src/NoForbiddenWords.elm

```diff
@@ -326,4 +326,9 @@ fastConcat =
 
 fastConcatMap : (a -> List b) -> List a -> List b
 fastConcatMap fn =
-    List.foldr (fn >> (++)) []
+    let
+        helper : a -> List b -> List b
+        helper item acc =
+            acc ++ fn item
```

Comment:
You have changed the order here. It should be `fn item ++ acc`

Reply:
Oh yeah! Fun thing though, none of the tests cared! :D